.travis/maybe-publish-coverage.sh

```diff
@@ -8,7 +8,7 @@ then
   exit 0
 fi
 
-if [ "$TRAVIS_NODE_VERSION" != "6.9.1" ]
+if [ "$TRAVIS_NODE_VERSION" != "8.1.2" ]
```

Comment:
Should we change this to `if [[ ! "${PUBLISH_NODE_VERSION:-8.1.2}" =~ ^$TRAVIS_NODE_VERSION ]]`, introduced in https://github.com/ciena-blueplanet/pr-bumper/pull/136?

Reply:
done